### DISCUSSION TITLE
State of the Map EU 2023

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#5294] [Ideas]
I am contemplating whether to go to the SotM EU 2023 in Antwerp on November 10th - 12th.

Anyone interested to go, too? I find a few talks interesting. 

We could rent a large holiday flat / AirBnb together in that case, and make this a bit of a StreetComplete developer meeting, too.

What would you say? Pinging some recent contributors - @Helium314 @FloEdelmann @matkoniecz @mnalis @peternewman @arrival-spring @tapetis 

The SotM EU 2023 schedule is here: https://stateofthemap.eu/schedule

**Comment by FloEdelmann** [comment]
Thanks for the ping! I'd love to meet you all in person and there would be no better way to do so than at the SotM EU. Unfortunately I don't have time that weekend :slightly_frowning_face:

**Comment by mnalis** [comment]
Nice initiative, @westnordost! Unfortunately, I won't be able to attend this year, but it does sound like fun, so hopefully there will another chance in the future

**Comment by arrival-spring** [comment]
Nice idea, and some of the talks do look interesting, but I can't attend unfortunately.

**Comment by matkoniecz** [comment]
I am interested, though I have not made any plans so far.

**Reply by westnordost** [reply]
Well, for me it's fine if we are only use two, too. On the weekend, I can look for some AirBnb or something. If in the meantime someone else would like to go to, even better!

Would you stay there some day(s) before or after too? It's quite the train ride (if you go by train, that is) from Krakow to Antwerp, after all.

**Comment by Helium314** [comment]
Sorry for the late reply, but most likely I won't have time on that weekend anyway...

**Comment by peternewman** [comment]
Sorry for taking a while to get back, it doesn't look like I can make it either unfortunately.

**Comment by westnordost** [comment]
Mateusz and me are going, by the way.

**Comment by westnordost** [comment]
After having returned from Antwerp with Mateusz, I realize that it was quite fun and would like to repeat that. The conference itself was fine too, but most of all, to just get to know you all.

Since nobody else could come to Antwerpen but seemed inclined in general, who would fancy a StreetComplete / OpenStreetMap hacking meeting some time in the winter?

We could rent a holiday home for a weekend or week in the countryside somewhere.  (Date, time and even location is obviously not set yet as it depends on who's coming. For winter, somewhere in snowy highlands seems like a nice place to be. First thought: Harz highlands or the Elbe Sandstone Mountains)

Mateusz and me did that in 2020. We rented a large holiday apartment in the countryside (Mecklenburg Lake Plateau) for a week. It was really fun, we cooked together, played games together, did several hiking, biking and canoeing excursions and of course, well, were a bit productive, too. 
The other people that were with us were friends of mine who actually used the opportunity to do home office in the countryside during this week and just took part in activities on the weekend and evenings. So, if you don't have time off work for that long, that might be an option for you, too.

Photos from last time:
![20200906_192124](https://github.com/streetcomplete/StreetComplete/assets/4661658/c843c33b-a1e8-4c9d-9410-6337f5bdfe5e)
![20200901_192049](https://github.com/streetcomplete/StreetComplete/assets/4661658/e1dda578-68bb-451b-a300-4f9831a918a7)


So, what do you think? Who's interested?

**Reply by FloEdelmann** [reply]
That sounds great! I'd certainly be interested :slightly_smiling_face:

**Reply by matkoniecz** [reply]
I am interested! And if it turn out to be too late to find good times for winter - I would be happy about Summer trip :)

**Reply by westnordost** [reply]
Cool, three is already a crowd! I also asked a friend of mine (who joined last time also, he was the "actually I am doing home office"-guy)

Let's find a date! 

Others who so far didn't reply could also add themselves there if they are interested! I'll follow up with an email once at least those added themselves of whom I know would like to join:

https://framadate.org/RTHmt7cUHxHFh4LR

**Reply by matkoniecz** [reply]
@westnordost 

"week Feb 5th..." would be covering availability on 5th - 11th, right?

**Reply by westnordost** [reply]
@FloEdelmann you did not put yourself into the framadate yet

**Reply by FloEdelmann** [reply]
Sorry, thanks for bringing this back to my attention. Seems like I forgot to click the save button...

**Reply by westnordost** [reply]
Nice, thanks. I sent an email to all who answered. For anyone else who wants to join, make up your mind quickly, we will book the accommodation soon.

**Reply by Helium314** [reply]
It's a little late, but since I received a nice reminder, the location and info that there is still space... I'll join too.
